rules/rules.js

```diff
@@ -209,7 +237,7 @@ const JSRule = function (ruleConfig) {
 
   const doExecute = function (module, input) {
     try {
-      return ruleConfig.execute(getTriggeredData(input));
+      return ruleConfig.execute(getTriggeredData(input), input, module);
```

Comment:
I see #117 needs the command from the input, should we modify getTriggeredData to return an object that has the needed command as a string member?  I'm always a bit hesitant to overload calls like this if we don't have to ( the api starts to bloat)